### PR TITLE
Adding tests for datasource setup.

### PR DESCRIPTION
**Pull request by antip00**

### CODE REVIEW

File: docker/clickhouse/options_method_test.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<clickhouse>
+    <http_options_response>
+        <header>
+            <name>Access-Control-Allow-Origin</name>
+            <value>http://grafana:3000</value>
```
Comment (Slach) [inline docker/clickhouse/options_method_test.xml:5]: let's just change to `*` and use `options_method.xml`

**Review by Slach — CHANGES_REQUESTED** [review]

**Comment by Slach** [comment]
also try multiple item for the same header with following format

```xml
        <header>
            <name>Access-Control-Allow-Origin</name>
            <value>http://localhost:3000</value>
        </header>
        <header>
            <name>Access-Control-Allow-Origin</name>
            <value>http://grafana:3000</value>
        </header>

```

File: docker/clickhouse/options_method_test.xml (hunk 1 above)
Comment (antip00) [reply docker/clickhouse/options_method_test.xml:5]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSNotSupportingCredentials?utm_source=devtools&utm_medium=firefox-cors-errors&utm_campaign=default
Screenshot from 2024-06-27 14-41-22

**Comment by antip00** [comment]
> also try multiple item for the same header with following format
> 
> ```
>         <header>
>             <name>Access-Control-Allow-Origin</name>
>             <value>http://localhost:3000</value>
>         </header>
>         <header>
>             <name>Access-Control-Allow-Origin</name>
>             <value>http://grafana:3000</value>
>         </header>
> ```
Screenshot from 2024-06-27 14-45-04
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMultipleAllowOriginNotAllowed?utm_source=devtools&utm_medium=firefox-cors-errors&utm_campaign=default

**Review by Slach — APPROVED** [review]

**Pull request merged**